programs/bpf_loader/src/lib.rs

```diff
@@ -258,14 +270,15 @@ impl Executor for BPFExecutor {
             };
 
             log!(logger, "Call BPF program {}", program.unsigned_key());
-            let instruction_meter = ThisInstructionMeter::new(compute_meter.clone());
+            let mut instruction_meter = ThisInstructionMeter::new(compute_meter.clone());
             let before = compute_meter.borrow().get_remaining();
-            let result = vm.execute_program_metered(
-                parameter_bytes.as_slice(),
-                &[],
-                &[heap_region],
-                instruction_meter,
-            );
+            let result = if is_jit_enabled {
+                // TODO: Cache JIT compiled code
+                vm.jit_compile().unwrap();
```

Comment:
this goes away when the executable contains the jit compiled code?

Reply:
Do you mean the `.jit_compile()` or the `.unwrap()` ?

Reply:
Both, but good point, replace this `unwrap` with a friendly error